test/governance/extensions/GovernorWithParams.test.js

```diff
@@ -120,7 +120,7 @@ contract('GovernorWithParams', function (accounts) {
         expect(votes.forVotes).to.be.bignumber.equal(weight);
       });
 
-      describe('voting by signature', function () {
+      describe.only('voting by signature', function () {
```

Comment:
we should have a CI check for that.

Reply:
> we should have a CI check for that.

We do have the codecov check that fails whenever this happens because the testing diff for this file suddenly drops, so nothing to worry :p

Removed